src/sentry/similarity/backends/redis.py

```diff
@@ -38,7 +37,7 @@ def _build_signature_arguments(self, features):
 
         arguments = []
         for bucket in band(self.bands, self.signature_builder(features)):
-            arguments.extend([1, ",".join(map("{}".format, bucket)), 1])
+            arguments.extend([1, ",".join(str(b) for b in bucket), 1])
```

Comment:
```suggestion
            arguments.extend([1, ",".join(f"{b}" for b in bucket), 1])
```

Any reason to not do that?

Reply:
the `str` call is a little more explicit about what's going on

Reply:
Dang, thought you would be on the `f"{s}"` boat, lol